### PR TITLE
Parsing vcs urls in requirements and zope-event fix

### PR DESCRIPTION
**Pull request by damianwasik98**
Description in commit msgs ☺️

### CODE REVIEW

**Comment by matthias-bach-by** [comment]
Thanks for your contribution. I'll try to get this reviewed over the next days.

File: core-requirements.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ wheel-filename
 wheel-inspect>=1.6.0
 pip>=1.5.3
 junit-xml
+pip-requirements-parser==32.0.1
```
[the remark this answers is not in the document]
Comment (damianwasik98) [reply core-requirements.txt]: It is just a habbit from some bigger project. I am used to pin exact version of dependencies in order to have more control and some kind of prevention from unexpected bugs on production.

I have just changed it to minimum version 😀

File: devpi_builder/wheeler.py [hunk 2]
```diff
@@ -64,6 +68,7 @@ def _find_wheel(self, name, version):
         Find a wheel with the given name and version
         """
         candidates = [WheelFile(filename) for filename in glob.iglob(path.join(self.wheelhouse, '*.whl'))]
+        name = self._standardize_package_name(name)
```
[the remark this answers is not in the document]
Comment (damianwasik98) [reply devpi_builder/wheeler.py:71]: I cherry-picked your commits. Thank you 😉

**Review by matthias-bach-by — APPROVED** [review]

**Comment by damianwasik98** [comment]
Hi @matthias-bach-by 😉 Are you planning to release a new version of package to pypi with this feature?

**Comment by matthias-bach-by** [comment]
The new version is published at https://pypi.org/project/devpi-builder/6.1.0/. Thanks for your contribution!

**Pull request merged**